command/csi_volume.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type CSIVolumeCommand struct {
+	Meta
+}
+
+func (c *CSIVolumeCommand) Help() string {
+	helpText := `
+Usage: nomad csi volume <subcommand> [options]
+
+  csi volume groups all the CSI (Container Storage Interface) commands that
```

Comment:
If you compare this text to some of the other commands, there's some style issues. These notes apply to most of the help text in this PR:
- I think the style we use is "imperative mood, simple present tense", but my formal grammar skills are a little rusty so I might be wrong with the exact names. But compare to `operator` or `job`, etc. and you'll see what I mean. This line is fine as `Groups all...` but some of the others are passive voice `... is used...`
- It looks like we usually start these without repeating the command name, so `Groups all...`.  
- We capitalize `Nomad` when it isn't the CLI, so `... available to a Nomad task.`